### PR TITLE
Added working login endpoint

### PR DESCRIPTION
**Pull request by soderdrew**
#5 Created login endpoint and fixed error with UserNotFound

### CODE REVIEW

File: backend/controllers/UserController.ts [hunk 1]
```diff
@@ -1,15 +1,19 @@
+import { ObjectId } from 'mongoose';
```
Comment (Darrick-Oliver) [inline backend/controllers/UserController.ts:1]: No need for this import, same with L9-L10

**Review by Darrick-Oliver — CHANGES_REQUESTED** [review]
Everything else looks good

File: backend/controllers/UserController.ts (hunk 1 above)
Comment (soderdrew) [reply backend/controllers/UserController.ts]: Good catch teammate! This has been resolved.

**Comment by Darrick-Oliver** [comment]
LGTM

**Review by Darrick-Oliver — APPROVED** [review]

**Pull request merged**